apps/dashboard/src/api/environments.ts

```diff
@@ -0,0 +1,14 @@
+import type { IEnvironment } from '@novu/shared';
+import { get } from './api.client';
+
+export async function getEnvironments() {
+  const { data } = await get<{ data: IEnvironment[] }>('/environments');
+
+  return data;
+}
+
+export async function getCurrentEnvironment() {
```

Comment:
This endpoint will be deprecated from the backend so please remove this function. Environment selection should be a pure FE concern.

Reply:
ahh I saw somewhere your comment saying to use the `/environments/me` endpoint instead of these calculations, but I can change that.

Reply:
Please send me the link. I might be missing something.

Reply:
https://github.com/novuhq/novu/blob/7b6e0e276c7b734ab19b09a2a63e8e5106e5ea13/apps/web/src/components/providers/EnvironmentProvider.tsx#L45